crates/typst/src/math/accent.rs

```diff
@@ -63,6 +117,29 @@ pub struct AccentElem {
     pub size: Smart<Rel<Length>>,
 }
 
+// Keep it synced with the documenting table above.
+accents! {
```

Comment:
I see how it's nice to have this close to the table, but I think I'd still prefer for it to be in its canonical place, below `impl Accent`.